traefik/imageroot/systemd/user/certificate-exporter.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Executes exporter on acme.json changes
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/nethserver/agent.env
+EnvironmentFile=%S/state/environment
+EnvironmentFile=%S/state/agent.env
+ExecStart=/usr/local/bin/certificate-exporter -v %h/.local/share/containers/storage/volumes/traefik-acme/_data/acme.json ${MODULE_ID}
```

A system-wide helper script shouldn't be needed. I'd prefer to have a module-local script, e.g.

```suggestion
ExecStart=%S/bin/export-certificate
WorkingDirectory=%h/.local/share/containers/storage/volumes/traefik-acme/_data/
```

The local script implementation can expect to run on the right directory with the right environment.

The above `EnvironmentFile`s should already import the variables used by the script:
- NODE_ID
- MODULE_ID (I'd suggest also AGENT_ID, as it is already prefixed like `module/traefik1`)